packages/webview_flutter/webview_flutter_android/lib/webview_android.dart

```diff
@@ -65,5 +65,6 @@ class AndroidWebView implements WebViewPlatform {
   }
 
   @override
-  Future<bool> clearCookies() => MethodChannelWebViewPlatform.clearCookies();
+  Future<bool> clearCookies() =>
+      WebViewCookieManagerPlatform.instance!.clearCookies();
```

Comment:
Should include a `null` check. The bang operator (`!`) should only be used if it is clear in the same code block that the field is set.